splice_spark/src/main/spark2.2/com/splicemachine/spark/splicemachine/SplicemachineContext.scala

```diff
@@ -45,20 +45,25 @@ import org.apache.hadoop.hbase.util.Bytes
 import org.apache.spark.api.java.JavaRDD
 import org.apache.spark.scheduler.{SparkListener, SparkListenerApplicationEnd}
 
-object Holder extends Serializable {
+private object Holder extends Serializable {
   @transient lazy val log = Logger.getLogger(getClass.getName)
 }
 
 /**
   *
   * Context for Splice Machine.
   *
-  * @param options
+  * @param options Supported options are SpliceJDBCOptions.JDBC_URL, SpliceJDBCOptions.JDBC_INTERNAL_QUERIES, SpliceJDBCOptions.JDBC_TEMP_DIRECTORY
   */
 class SplicemachineContext(options: Map[String, String]) extends Serializable {
+  private[this] val url = options.get(JDBCOptions.JDBC_URL).get
 
-  val url = options.get(JDBCOptions.JDBC_URL).get
-  
+  /**
+    *
+    * Context for Splice Machine, specifying only the JDBC url.
+    *
+    * @param url JDBC Url with authentication parameters
```

Comment:
Why not to reference `JDBCOptions.JDBC_URL` so the doc is automatically copied / accessible?